tests/text_processing/tickets_data/PHPBB3-13921.html

```diff
@@ -0,0 +1 @@
+<span style="font-size: 200%; line-height: normal"></span><div style="text-align:center"><span style="font-size: 200%; line-height: normal">xxx</span></div>
```

Comment:
Those files contain the data that corresponds to a normal post and its output. To have a LF here you'd need a LF in the original text, but then it would add a `<br>` to the output too and trailing whitespace is normally removed anyway.